base/c.jl

```diff
@@ -4,6 +4,11 @@
 
 import Core.Intrinsics: cglobal, bitcast
 
+# Equivalent to Sys.iswindows(), but that's not yet defined
+const _WINDOWS = let k = ccall(:jl_get_UNAME, Any, ())
```

Comment:
Kind of annoying to give this a name. I think we have control over `jl_get_UNAME` so it should only ever return a single value